src/Database/LSMTree/Common.hs

```diff
@@ -112,13 +109,19 @@ instance IOLike IO
 type Session :: (Type -> Type) -> Type
 type Session = Internal.Session'
 
-{-# SPECIALISE withSession :: Tracer IO Internal.LSMTreeTrace -> HasFS IO HandleIO -> HasBlockIO IO HandleIO -> FsPath -> (Session IO -> IO a) -> IO a #-}
+{-# SPECIALISE withSession ::
+     Tracer IO Internal.LSMTreeTrace
+  -> HasFS IO HandleIO
+  -> HasBlockIO IO HandleIO
+  -> FsPath
+  -> (Session IO -> IO a)
+  -> IO a #-}
```

Comment:
Wrapping all the `SPECIALISE` pragmas wasn't really necessary, but let's keep it in